chart/k8gb/templates/external-dns/external-dns-ns1.yaml

```
@@ -15,6 +15,10 @@ spec:
         app: external-dns-ns1
     spec:
       serviceAccountName: external-dns
+      securityContext:
```

How about making securityContext parameterized though `values.yaml` file, so it can be configured and reused in all external-dns deployments?
```suggestion
      securityContext:
{{- toYaml .Values.externaldns.securityContext | nindent 8 }}
```

---

@somaritane good idea, implemented, please check